.vscode/tasks.json

```diff
@@ -83,6 +67,20 @@
         "problemMatcher": "$msCompile",
         "group": "build"
     },
+    {
```

Comment:
I removed the `build.sh/cmd` version because it was not disabling console log correctly. At that point it was more closely matched with `"build Roslyn.sln` so moved it below that.